lib/fog/compute/terremark/models/tasks.rb

```diff
@@ -24,7 +24,7 @@ def task_list_id
           organization = service.get_organization(service.default_organization_id).body
           @task_list_id ||=
               if service.default_organization_id && organization
-                organization['Links'].find {|link| link['type'] == 'application/vnd.vmware.vcloud.tasksList+xml'}['href'].split('/').last.to_i
+                organization["Links"].find {|link| link["type"] == "application/vnd.vmware.vcloud.tasksList+xml"}["href"].split("/").last.to_i
```

Comment:
Prefer `detect` over `find`.<br>Space between { and | missing.<br>Space missing inside }.